lib/HttpClient.js

```diff
@@ -16,7 +16,7 @@ var util = require('util');
 var assert = require('assert-plus');
 var backoff = require('backoff');
 var mime = require('mime');
-var once = require('once');
+var once = require('once').strict;
```

Comment:
nit: I think I'd find it more obvious what behavior is used by each "onced" callback if each was explicitly created with `once.strict` below instead of `once`. Reviewing this PR I got confused: I did not understand what had changed since the call sites were using `once`: I thought they were still using the non-strict versions.

This is highly subjective though, I'm fine if you prefer this style!